### PR TITLE
Add read more button to the aid package card

### LINKED ISSUE #347: Update the aid package card
Update the aid package card on the donor portal as follows:

Screenshot 2022-08-04 at 12 59 39

Use the following image (compress the image to a lower res please):
the-lore-com-rRBtKW_aw_U-unsplash

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #347 

## Goals

- Trimming the long descriptions

## Approach

- Trimmed the description at 300 character limit
- Add the read more link
- Add the aid package image
### Screenshots

image

### CODE REVIEW

File: donor-portal/src/pages/donate-now/styles.css [hunk 1]
```diff
@@ -27,14 +27,18 @@
...
+
 .package-card {
   display: flex;
   margin-bottom: 2rem;
   border: 1px solid var(--elixir-color-gray-500);
 }
 
 .card-details {
-  width: 100%;
+  width: 70%;
```
Comment (jayasanka-sack) [inline donor-portal/src/pages/donate-now/styles.css:41]: This should not be from percentages. Provide image size from pixels and let `card-details` take the rest of the space. 

Use a `flex`.

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks for the PR @anjula-sack !

I added a comment. 

Also, Could you please remove the space between title and the description?

File: donor-portal/src/pages/donate-now/styles.css (hunk 1 above)
Comment (anjula-sack) [reply donor-portal/src/pages/donate-now/styles.css]: sure

**Comment by anjula-sack** [comment]
localhost_3000_donate-now (2)

@jayasanka-sack

**Review by jayasanka-sack — APPROVED** [review]
Thanks @anjula-sack !

**Pull request merged**